### PR TITLE
Android structure

### PR DESCRIPTION
**Pull request by Saadnajmi**
I added 5 activities corresponding to the main features of the client app.

I also added buttons to navigate to them. The next step is making a navigation drawer.

### CODE REVIEW

File: mobile/Scorekeeper/Scorekeeper.Android/Scorekeeper.Android.csproj.bak [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zuhairp) [inline mobile/Scorekeeper/Scorekeeper.Android/Scorekeeper.Android.csproj.bak]: Remove this file from the branch

File: mobile/Scorekeeper/Scorekeeper.iOS/Scorekeeper.iOS.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (zuhairp) [inline mobile/Scorekeeper/Scorekeeper.iOS/Scorekeeper.iOS.csproj]: Is this file related to this changeset? If not, remove it from the PR

File: mobile/Common/Common.Android/Common.Android.csproj.bak [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zuhairp) [inline mobile/Common/Common.Android/Common.Android.csproj.bak]: Remove this file from the branch

File: mobile/Client/Client.iOS/Client.iOS.csproj [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (zuhairp) [inline mobile/Client/Client.iOS/Client.iOS.csproj]: Is this file relevant to the changeset? If not, remove it from the PR

File: mobile/Client/Client.Android/Client.Android.csproj.bak [hunk 5]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zuhairp) [inline mobile/Client/Client.Android/Client.Android.csproj.bak]: Is this file relevant to the changeset? If not, remove it from the PR

**Comment by zuhairp** [comment]
Can you attach some screenshots of how this looks?

**Comment by Saadnajmi** [comment]
It's easier if I delete this branch and start over, considering we deleted all the android projects
